test-apps/display-test-app/src/frontend/Viewer.ts

```diff
@@ -451,7 +451,7 @@ export class Viewer extends Window {
   }
 
   private async closeIModel(): Promise<void> {

```

Comment:
Could this just be this._imodel.close() now?

Reply:
made that change.